src/reflect/scala/reflect/internal/Definitions.scala

```diff
@@ -1103,6 +1103,7 @@ trait Definitions extends api.StandardDefinitions {
     lazy val BridgeClass                = requiredClass[scala.annotation.bridge]
     lazy val ElidableMethodClass        = requiredClass[scala.annotation.elidable]
     lazy val ImplicitNotFoundClass      = requiredClass[scala.annotation.implicitNotFound]
+    lazy val ImplicitAmbiguousClass     = requiredClass[scala.annotation.implicitAmbiguous]
```

Comment:
Better to use `getClassIfDefined`, so the new complier doesn't crash if used with an older library, as can happen in some IDE use cases.